to.etc.domui/src/main/java/to/etc/domui/component/layout/TabPanelBase.java

```diff
@@ -382,13 +183,23 @@ public void closeTab(@Nonnull final ITabHandle th) throws Exception {
 		m_tablist.remove(index);
 
 		if(isBuilt()) {
-			ti.getTab().remove();
-			ti.getSeparator().remove();
-			ti.getContent().remove();
+			Li tab = ti.getTab();
+			if(tab != null)
+				tab.remove();
+
+			Li seperator = ti.getSeparator();
```

Comment:
Spelling error in separator, you might want to switch on spelling check in IntelliJ; it is quite nice ;)